src/drt/src/TritonRoute.tcl

```diff
@@ -310,6 +310,30 @@ proc pin_access { args } {
   }
   drt::pin_access_cmd $db_process_node $bottom_routing_layer $top_routing_layer $verbose $min_access_points
 }
+sta::define_cmd_args "check_drc" {
+    [-drc_box box]
+    [-output_drc filename]
```

Comment:
You don't need to repeat drc in the option names.  -box & -output_file are fine.

Put the command in the drt namespace to make it private.

Reply:
Why do you want it private?

Reply:
I don't want to create false expectations that this is a general purpose drc engine.  It is a sufficient engine to drive the router and no more (ie it doesn't check violations between fixed shapes).

Reply:
@maliberty  ok. done